eng/tools/rush-runner.js

```diff
@@ -22,6 +22,25 @@ const reducedDependencyTestMatrix = {
     '@azure/test-utils-perfstress',
     '@azure-tools/test-recorder',
     '@azure/synapse-monitoring'
+  ],
+  'test-utils': ['@azure-rest/core-client',
+    '@azure-rest/core-client-lro',
+    '@azure-rest/core-client-paging',
+    '@azure-rest/purview-account',
+    '@azure-tests/perf-storage-blob',
+    '@azure/ai-text-analytics',
+    '@azure/arm-compute',
+    '@azure/dev-tool',
+    '@azure/identity',
+    '@azure/identity-cache-persistence',
+    '@azure/identity-vscode',
+    '@azure/service-bus',
+    '@azure/storage-blob',
+    '@azure/template',
+    '@azure/test-utils',
+    '@azure/test-utils-perfstress',
+    '@azure-tools/test-recorder',
+    '@azure/synapse-monitoring'
   ]
```

Comment:
```suggestion
  'test-utils': [
    '@azure-rest/purview-account',
    '@azure/arm-eventgrid',
    '@azure/ai-text-analytics',
    '@azure/arm-compute',
    '@azure/identity',
    '@azure/identity-cache-persistence',
    '@azure/identity-vscode',
    '@azure/storage-blob',
    '@azure/template',
    '@azure/test-utils',
    '@azure/test-utils-perfstress',
    '@azure-tools/test-recorder',
    '@azure-tools/test-recorder-new',
    '@azure/synapse-monitoring'
  ]
```

Reply:
We don't need core packages.
Adding 
- test-utils
- perf framework
- recorder
- recorder-new
- 1 azure-rest package
- 1 core-v1 perf-project
- 1 core-v2 perf project
- 1 arm package that uses recorder
- template
- identity - all packages
- 1 core-v1 package
- 1 core-v2 package